packages/xo-web/src/xo-app/self/index.js

```diff
@@ -696,12 +700,26 @@ class ResourceSet extends Component {
 
 const compareName = (a, b) => (a.name < b.name ? -1 : 1)
 
+@addSubscriptions({ resourceSets: subscribeResourceSets })
+@connectStore({ areObjectsFetched })
```

Comment:
What about creating a dedicated selector to resolve the resource sets instead of checking areObjectsFetched everywhere we use resource sets?